pyviz_comms/__init__.py

```diff
@@ -30,6 +30,10 @@ def _jupyter_labextension_paths():
 except NameError:
   get_ipython = None
 
+# Setting this so we can check the launched jupyter has pyviz_comms installed
+if not (get_ipython and sys.argv[0].endswith('ipykernel_launcher.py')):
```

Comment:
`ipykernel_launcher.py` file is what launches a notebook. 

I will update the comments if we decide to proceed with the PR. 

The name of the environment variable can change maybe to `PYVIZ_COMMS_INSTALLED`.